lib/script_base.rb

```diff
@@ -54,6 +55,9 @@ def run
 
     stderr.puts "*Task*: `#{result.subtask}`"
     stderr.puts "*UUIDs*: #{result.uuids.map { |uuid| "`#{uuid}`" }.join(', ')}"
+    if result.messages && result.messages.length > 0
```

Comment:
personally I am a big fan of [ActiveSupport's `#present?`](https://guides.rubyonrails.org/active_support_core_extensions.html#blank-questionmark-and-present-questionmark) which checks that arrays are non-empty (and works on `nil` values as well)

```suggestion
    if result.messages.present?
```

Reply:
Implemented in forthcoming commit